### PR TITLE
Rollbar integration

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

**Comment by AronNovak** [comment]
This is the 1-1 port from client project. We can improve it as agreed in a follow-up.

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ stages:
 env:
   global:
   - DDEV_NO_INSTRUMENTATION=true
+  - ROLLBAR_SERVER_TOKEN=df6ce617465b4980afdecc95ed1b42de
   - PANTHEON_GIT_URL="ssh://codeserver.dev.5acc1750-abf0-47ef-a6d8-6890a6efaca8@codeserver.dev.5acc1750-abf0-47ef-a6d8-6890a6efaca8.drush.in:2222/~/repository.git"
```
Comment (amitaibu) [inline .travis.yml:16]: Off topic - now with the Codespace integration, we can probably remove the Pantheon.

File: .travis.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ stages:
 env:
   global:
   - DDEV_NO_INSTRUMENTATION=true
+  - ROLLBAR_SERVER_TOKEN=df6ce617465b4980afdecc95ed1b42de
```
Comment (amitaibu) [inline .travis.yml:15]: Can we encrypt the token (and change this one, so it's not exposed)?

File: composer.patches.json [hunk 3]
```diff
@@ -32,6 +32,9 @@
     },
     "drupal/seckit": {
       "d.o. #3206728": "https://www.drupal.org/files/issues/2024-09-06/seckit-2_x-src-field-length-3206728-8.patch"
+    },
+    "drupal/rollbar": {
+      "memory issues": "patches/3432364-rollbar-vars.patch"
```
Comment (amitaibu) [inline composer.patches.json:37]: Is there a do issue?

**Review by amitaibu — COMMENTED** [review]
Thanks, a few comments

File: composer.patches.json (hunk 3 above)
Comment (AronNovak) [reply composer.patches.json:37]: https://www.drupal.org/node/3432364

File: .travis.yml (hunk 2 above)
Comment (AronNovak) [reply .travis.yml:15]: This is not a real token, a dummy one from the documentation. It is enough if we try to send an item to Rollbar, if it fails or not, for the sake of this test, it does not matter.

Comment (AronNovak) [reply .travis.yml:15]:
```
 aaron  ~  curl -H 'X-Rollbar-Access-Token: df6ce617465b4980afdecc95ed1b42de' 'https://api.rollbar.com/api/1/projects'
{
  "err": 1,
  "message": "invalid access token"
} aaron  ~  

```

File: .travis.yml (hunk 1 above)
Comment (AronNovak) [reply .travis.yml:16]: I guess it's more a PM question, if they need persistent demo stuff or not.
@isapisa @nora-novak

Comment (amitaibu) [reply .travis.yml:16]: I don't think it's needed.

File: .travis.yml (hunk 2 above)
Comment (amitaibu) [reply .travis.yml:15]: Thanks, let's add a comment above it's dummy

File: composer.patches.json (hunk 3 above)
Comment (amitaibu) [inline composer.patches.json:37]:
```suggestion
      "d.o. #3432364": "patches/3432364-rollbar-vars.patch"
```

**Comment by amitaibu** [comment]
Needs a re-roll

**Comment by amitaibu** [comment]
@AronNovak sorry, I merged a PR before, and it caused another conflict 😬

**Review by amitaibu — APPROVED** [review]

**Pull request merged**